files/en-us/web/css/css_selectors/index.html

```diff
@@ -66,9 +66,9 @@ <h2 id="Combinators">Combinators</h2>
  <strong>Syntax:</strong> <code><var>A</var> ~ <var>B</var></code><br>
  <strong>Example:</strong> <code>p ~ span</code> will match all {{HTMLElement("span")}} elements that follow a {{HTMLElement("p")}}, immediately or not.</dd>
  <dt><a href="/en-US/docs/Web/CSS/Adjacent_sibling_combinator">Adjacent sibling combinator</a></dt>
- <dd>The <code>+</code> combinator selects adjacent siblings. This means that the second element directly follows the first, and both share the same parent.<br>
+ <dd>The <code>+</code> combinator selects a single, immediately succeeding/following, sibling element. An element can only be adjacent to sibling (same parent) elements where no siblings exist between the two.<br>
```

Comment:
```suggestion
 <dd>The <code>+</code> combinator matches the second element only if it _immediately_ follows the first element.<br>
```
Thiis is close to what is in the specific page and avoid the word sibling that is complex. It also avoid the use of the `/` sign and of parenthesis that all are making the sentence difficult to read,